README.md

```diff
@@ -624,130 +650,155 @@ Note that it is also possible to generate a synthetic dataset based on the entir
+
+1. `simple` uses an SFT Trainer on Linux and MLX on MacOS. This type of training takes roughly an hour and produces the lowest fidelity model but should indicate if your data is being picked up by the training process.
+2. `full` uses a custom training loop and data processing functions for the granite family of models. This loop is optimized for CPU and MPS functionality. Please use `--pipeline=full` in combination with `--device=cpu` (Linux) or `--device=mps` (MacOS). You can also use `--device=cpu` on a MacOS machine. However, MPS is optimized for better performance on these systems.
+3. `accelerated` uses the instructlab-training library which supports GPU accelerated and distributed training. The `full` loop and data processing functions are either pulled directly from or based off of the work in this library.
 
 After running `ilab model train`, the output locations depend on the chosen pipeline or strategy:
 
 | **Pipeline/Strategy**              | **Operating System** | **Output Location/Details**                                                                                             |
 |------------------------------------|----------------------|--------------------------------------------------------------------------------------------------------------------------|
 | `simple`                           | Linux                | Model saved in `models` directory as `ggml-model-f16.gguf`.                                                              |
-|                                    | MacOS                | Model saved in `<model_name>-mlx-q` directory.                                                                           |
+| `simple`                           | MacOS                | Model saved in `<model_name>-mlx-q` directory.                                                                           |
 | `full`                             | Linux & MacOS        | `.bin` and `.gguf` models saved in `~/.local/share/instructlab/checkpoints/hf_format` directory. Two models in each `sample_*` directory: one quantized (`Q4-M-K` suffix) and one full precision. |
 | `accelerated`                      | Linux                | Models saved in `~/.local/share/instructlab/checkpoints`. Can be evaluated with `ilab model evaluate` to choose the best one. |
 | `lab-multiphase`                   | Linux                | Phase 1 models saved in `~/.local/share/instructlab/phased/phase1/checkpoints` (Knowledge training). Phase 2 models saved in `~/.local/share/instructlab/phased/phase2/checkpoints` (Skills training). Evaluation is run for both phases to identify the best checkpoint. |
 
+#### Train the model locally
+
+- Train the model with your synthetic data with the `ilab model train` command:
+
+   ```shell
+   ilab model train
+   ```
+
+   ⏳ This step can potentially take **several hours** to complete depending on your computing resources. Please stop `ilab model chat` and `ilab model serve` first to free resources.
+
```

Comment:
Might be helpful to recommend a spec. Moo deng was on a 64GB ram system and took around 24 hours, is that right @cdoern ? Can we throw out a stat like that so folks have a rough idea?

Reply:
Im thinking about making like a performance chart based on tests we've done in a separate doc.